subiquity/ui/views/tests/test_identity.py

```diff
@@ -41,6 +66,27 @@ def test_done_enabled_when_valid(self):
         view_helpers.enter_data(view.form, valid_data)
         self.assertTrue(view.form.done_btn.enabled)
 
+    async def test_username_validation_system_reserved(self):
+        view = self.make_view()
+        view.controller.validate_username.return_value = \
+            UsernameValidation.SYSTEM_RESERVED
+        view_helpers.enter_data(view.form, system_reserved)
+        await asyncio.wait(view.form.validation_task)
```

Comment:
whereupon we find that wait takes an iterable. so this would be more correct if it were await asyncio.wait({view.form.validation_task}) but it would be even better if it was just await view.form.validation_task